hw/ip/prim/prim.core

```diff
@@ -15,6 +15,8 @@ filesets:
       - lowrisc:prim:pad_wrapper
       - lowrisc:prim:prim_pkg
       - lowrisc:prim:clock_mux2
+      - lowrisc:prim:flop
+      - lowrisc:prim:flop_2sync
     files:
       - rtl/prim_clock_inverter.sv
```

Comment:
this is unrelated to this PR, but we should probably consider making this a `prim_generic_clock_inv` as well, WDYT?

Reply:
agreed.

Reply:
o yeah definitely. We will need it if anyone wants to operate on the other edge. 
i think we have `prim_ck_mux, prim_ck_buf`, any others we're missing?

Reply:
we also have `prim_clock_gating`. I provisioned the inverter in the internal technology wrappers. just need to align the opensource side. I'll do this sometime this week.